pkg/client/history/ddlmanager.go

```diff
@@ -0,0 +1,227 @@
+package history
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudquery/cq-provider-sdk/helpers"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+	"github.com/georgysavva/scany/pgxscan"
+	"github.com/hashicorp/go-hclog"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+const (
+	listHyperTables = `SELECT hypertable_name FROM timescaledb_information.hypertables WHERE hypertable_schema=$1 ORDER BY 1`
+
+	setChunkTimeInterval = `SELECT * FROM set_chunk_time_interval($1, INTERVAL '%d hour');`
+	dataRetentionPolicy  = `SELECT history.update_retention($1, INTERVAL '%d day');`
+
+	dropTableView   = `DROP VIEW IF EXISTS "%[1]s"`
+	createTableView = `CREATE VIEW "%[1]s" AS SELECT * FROM history."%[1]s" WHERE cq_fetch_date = find_latest('history', '%[1]s')`
+
+	schemaName = "history"
+)
+
+type DDLManager struct {
```

Comment:
why no just call this History?

Reply:
We'd have too much "history" things, ie. config-related stuff in history.go (same dir) and so on. And `history.History` doesn't look too good.

Reply:
Accepted